index.bs

```diff
@@ -8011,6 +8011,22 @@ The <dfn>Credential Parameters</dfn> is a JSON [=Object=] passed to the [=remote
                 </td>
                 <td>boolean</td>
             </tr>
+            <tr>
+                <td>|userName|</td>
+                <td>
+                    The {{PublicKeyCredentialUserEntity|user}}'s {{PublicKeyCredentialEntity/name}} associated to the credential.
+                    If unset, the value will default to the empty string.
```

Comment:
Hmm, thinking about ways in which `PublicKeyCredentialUserEntity.name` can be set, [`user.name` is a required value when calling `.create()`](https://w3c.github.io/webauthn/#dom-publickeycredentialentity-name), and [`signalCurrentUserDetails()` requires `name`](https://w3c.github.io/webauthn/#dom-currentuserdetailsoptions-name) be set as well. It feels like a potential foot gun to allow `user.name` to end up as an empty string when using the virtual authenticator API 🤔

Reply:
On the other hand, I assume that requiring that these values be set would make the WebDriver method(s) incompatible with callers that haven't yet updated to set them?

Reply:
We cannot make it required. But we can make it so that if present, it has to be set. If not present, it will be left unchanged.

This means that it is still possible to create a credential with an empty name through the virtual authenticator, but that was already a problem before this PR and nobody ever complained, so it's probably not a big deal.

Reply:
> if present, it has to be set

As in not `null`? That seems reasonable.

Reply:
Wait a minute, it's required but it can be empty :thinking: 

That means that the current text is fine. If left unset, it defaults to the empty string. RPs can make credentials with empty names through regular webauthn already.